recipes/cis_interface/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "cis_interface" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b815bbb0f9fa38d9e8f0f9e6a47bb4bfd67dbb9f5b6cdd81288dfe84a612e59a
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - czmq
+    - pip
+    - python
+    - sysv_ipc  # [unix]
+    - zeromq
+  run:
+    - cerberus
+    - czmq
+    - jsonschema
+    - matplotlib
+    - nose
+    - numpy
+    - pandas  # [not py34]
```

Comment:
please remove the ``# [not py34]`` because ``conda-forge`` is not creating packages for python 34 anymore. You can also delete the line ``33`` which is ``pandas < 0.21  # [py34]``